src/current/cockroachcloud/authorization.md

```diff
@@ -108,15 +117,45 @@ Cluster Administrators can perform all of the [Cluster Operator actions](#cluste
 - Cluster Administrators for the whole organization (rather than scoped to a single cluster) can [create new clusters](create-your-cluster.html).
 - Access the [DB Console](network-authorization.html#db-console).
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
 ### Cluster Creator
 
 Cluster Creators can create clusters in an organization. A cluster's creator is automatically granted the [Cluster Administrator](#cluster-administrator) role for that cluster upon creation.
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
 ### Cluster Developer
 
-Users with this role can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
+Cluster Developers can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
+
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
+### Folder Admin
```

Comment:
Another inconsistency between Admin/Administrator. I strongly prefer harmonizing this to one or the other. I also understand if this is inconsistent in the UI and we're at the mercy of what's in the product.

Reply:
@jramos-crl @abhinavg6 for product consideration. These are what the roles are called.

Reply:
@mdlinville we should have a bigger conversation of how we might align other roles, but given how the API name is FOLDER_ADMIN we should keep it as Folder Admin here in the docs too.